src/Language/PureScript/CoreFn/Desugar.hs

```diff
@@ -134,61 +135,64 @@ mkTypeClassConstructor ss com supers members =
 -- |
 -- Desugars expressions from AST to CoreFn representation.
 --
-exprToCoreFn :: Environment -> Maybe SourceSpan -> [Comment] -> Maybe Type -> A.Expr -> Expr Ann
-exprToCoreFn _ ss com ty (A.NumericLiteral v) =
+exprToCoreFn :: ModuleName -> Environment -> Maybe SourceSpan -> [Comment] -> Maybe Type -> A.Expr -> Expr Ann
```

Comment:
Again, not essential, but I see a lot of these types of things in the codebase generally. I think we should try to close over constant things like this using `where` clauses to tidy things up.